cmd/collectors/zapi/README.md

```diff
@@ -1,167 +1,50 @@
+## Configuration
 
-Below we explain how to create an ONTAP user and role for Harvest using ONTAP System Manager (Classic interface & New interface) and CLI.
+The parameters and configuration are similar to those of the [ZapiPerf collector](../zapiperf/README.md). Therefore, only what is different will be discussed here.
 
-#### System Manager: New interface
+### Collector configuration file
 
-*Note: in this section we add a user with read-access to all API objects. For limited access, use either the classic interface or the CLI*
+Parameters different from ZapiPerf:
 
 
-Open System Manager. Click on *CLUSTER* in the left menu bar, *Settings* and *Users and Roles*.
-<center><img src="examples/ontap_user_sm_0.png" width="75%"></center>
+| parameter              | type         | description                                      | default                |
+|------------------------|--------------|--------------------------------------------------|------------------------|
+| `schedule`             | required     | same as for ZapiPerf, but only two elements: `instance` and `data` (collector does not run a `counter` poll) ||
+| `no_max_records`       | bool, optional | don't add `max-records` to the ZAPI request    |                        |
+| `collect_only_labels`  | bool, optional | don't look for numeric metrics, only submit labels  (suppresses the `ErrNoMetrics` error)| |
+| `only_cluster_instance` | bool, optional | don't look for instance keys and assume only instance is the cluster itself ||
 
-In the right column, under *Roles*, click on *Add* to add a new role.
-<center><img src="examples/ontap_user_sm_1.png" width="75%"></center>
 
-Choose a role name (e.g. *harvest2-role*). In the *REST API PATH* field type */api* and select *Read-Only* as for *ACCESS*. Click on *Save*. 
+### Object configuration file
 
-<center><img src="examples/ontap_user_sm_2.png" width="75%"></center>
+The Zapi collector does not have the parameters `instance_key` and `override` parameters.
 
-In the left column, under *Users*, click on *Add* to create a new user. Choose a username. Under *Role*, select the role that we just created. Under *User Login Methods* select *ONTAPI*, and select one of the two authentication methods. Type in a password if you chose *Password*. Click on *Save*
-<center><img src="examples/ontap_user_sm_3.png" width="75%"></center>
+#### `counters`
 
-If you chose *Password*, you can add the username and password to the Harvest configuration file and start Harvest. If you chose *Certificate* jump to [Using Certificate Authentication](#using-certificate-authentication) to generate certificates files.
+This section should contain the complete or partial attribute tree of the API that is queried. Since the collector does get counter metadata from the ONTAP system, two additional symbols are used for non-numeric attributes:
```

Comment:
This section contains the complete or partial attribute tree of the queried API.